package.json

```diff
@@ -14,11 +14,9 @@
     "serve:debug": "node --inspect=0.0.0.0:9229 --max_old_space_size=8192 dist/index.js",
     "start": "yarn serve",
     "start:debug": "nodemon -e ts -w ./src -x yarn serve:debug",
-    "gen-sql": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:generate -n",
-    "migrate": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:run",
-    "gen-module": "IASQL_ENV=local ./.module_migration.sh",
-    "docker-compose:debug": "IASQL_ENV=local docker-compose -f docker-compose.yml -f docker-compose.debug.yml up --build",
-    "docker-compose": "IASQL_ENV=local docker-compose up --build",
```

Comment:
Please restore these two.

Reply:
I had talked with Alejandro about removing them last week to avoid dirtying metrics for local usage, but had never made the PR

Reply:
I use `yarn docker-compose` *all* the time. I'd prefer we fix the metrics issue rather than remove it, because I'd just run it, anyways, when testing locally?

Reply:
It is not a metrics issue, we simply have to turn telemetry off by using `IASQL_ENV=ci docker-compose up —build` or a command that wraps it. The commands are restored anyway